dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -198,3 +205,9 @@ http.proxyredirect.httpsPort.desc=The (secure) port to redirect to. Incoming HTT
 
 http.autoDecompression=Auto decompression
 http.autoDecompression.desc=Specifies whether the HTTP Channel automatically decompresses incoming request body data.
+
+http.compression.types=List of content types for response compression
+http.compression.types.desc=Specifies a list of content types for which the HTTP Channel will perform compression of the response if able.
+
+http.compression.serverPreferredAlgorithm=Server preferred algorithm for compression
```

Comment:
**change**
Server preferred algorithm for compression
**to**
Server-preferred algorithm for compression